app/js/views/main.js

```diff
@@ -214,62 +206,56 @@ export default class MainView extends View {
     return mainViewTemplate;
   }
 
-  _addChildViews() {
-    // We need all of these things for the customizer to work
-    // But we don't want them sitting in the document tree when the
-    // customizer is closed, so we add them all only when we open.
-    this.container.appendChild(this.customizer); // the <fxos-customizer>
-    this.el.appendChild(this.childViews); // The highlighter and child views
+  _addToBody() {
```

Comment:
Are these functions really needed if they only have 1 callsite each? We should just inline them instead.